src/app/pages/psbt-request/components/psbt-decoded-request/psbt-decoded-request-node/psbt-decoded-inputs.tsx

```diff
@@ -0,0 +1,63 @@
+import * as btc from '@scure/btc-signer';
+import { Box, Text } from '@stacks/ui';
+import { truncateMiddle } from '@stacks/ui-utils';
+
+import { isUndefined } from '@shared/utils';
+
+import { satToBtc } from '@app/common/money/unit-conversion';
+import { useFiatValue } from '@app/pages/psbt-request/hooks/use-fiat-value';
+import { useOrdinalsAwareUtxoQueries } from '@app/query/bitcoin/ordinals/ordinals-aware-utxo.query';
+
+import { PsbtDecodedNodeLayout } from './psbt-decoded-node.layout';
+import { PsbtInputWithInscription } from './psbt-input-with-inscription';
+import { PsbtPlaceholderNode } from './psbt-placeholder-node';
+
+interface PsbtDecodedInputsProps {
+  addressNativeSegwit: string;
+  addressTaproot: string;
+  inputs: btc.TransactionInputRequired[];
+  showPlaceholder: boolean;
+}
+export function PsbtDecodedInputs({
+  addressTaproot,
+  addressNativeSegwit,
+  inputs,
+  showPlaceholder,
+}: PsbtDecodedInputsProps) {
+  const unsignedUtxos = useOrdinalsAwareUtxoQueries(inputs).map(query => query.data);
+  const getFiatValue = useFiatValue();
+
+  return (
+    <Box background="white" borderTopLeftRadius="16px" borderTopRightRadius="16px" p="loose">
+      <Text fontWeight={500}>Inputs</Text>
+      {showPlaceholder ? (
+        <PsbtPlaceholderNode />
+      ) : (
+        unsignedUtxos.map(utxo => {
+          if (isUndefined(utxo) || !utxo.address) return null;
+
+          const isInputCurrentAddress =
+            utxo.address === addressNativeSegwit || utxo.address === addressTaproot;
+          const inputValue = satToBtc(utxo.value).toString();
+          const fiatValue = getFiatValue(inputValue);
```

Comment:
Why not pass `utxo.value` here? 

Currently it goes from integer --> string decimal --> number decimal --> and back to integer in the hook.